src/interfaces/pyo3/py_cover_crypt.rs

```diff
@@ -209,14 +209,14 @@ impl CoverCrypt {
     /// - `access_policy_str`   : access policy
     /// - `public_key`          : CoverCrypt public key
     /// - `additional_data`     : additional data to encrypt with the header
-    /// - `authenticated_data`  : authenticated data to use in symmetric encryption
+    /// - `authentication_data`  : authentication data to use in symmetric encryption
     pub fn encrypt_header(
         &self,
         policy: &Policy,
         access_policy_str: &str,
         public_key: &PublicKey,
         additional_data: Option<Vec<u8>>,
-        authenticated_data: Option<Vec<u8>>,
+        authentication_data: Option<Vec<u8>>,
```

Comment:
Use an option of a slice here too ?
```suggestion
        authentication_data: Option<&[u8]>,
```